src/ui/input/input_wrap.jsx

```diff
@@ -55,7 +55,7 @@ InputWrap.propTypes = {
     PropTypes.arrayOf(PropTypes.element).isRequired
   ]),
   focused: PropTypes.bool,
-  invalidHint: PropTypes.string,
+  invalidHint: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
you can use just `PropTypes.node` here, because it already includes strings

Reply:
oh, that's great, thanks for the tip. I'll update.

Reply:
```suggestion
  invalidHint: PropTypes.node,
```